src/main/java/lesson02/ConsoleInput.java

```diff
@@ -0,0 +1,21 @@
+package lesson02;
+
+import java.util.Scanner;
+
```

Comment:
When first looking at this class during the review I wasn't able to identify that it is a utility class for some other task, think how this concern can be addressed within class/package naming?

Reply:
Utility classes moved to a new subpackage. Comments also added